src/metric-engine/src/engine/alter.rs

```diff
@@ -54,37 +53,52 @@ impl MetricEngineInner {
             let (region_id, request) = requests.pop().unwrap();
             self.alter_physical_region(region_id, request).await?;
         } else {
-            self.alter_logical_regions(requests, extension_return_value)
-                .await?;
+            let grouped_requests =
```

Comment:
Maybe we can add a fast path when region number is 1.

Reply:
Resolved in [4467745](https://github.com/GreptimeTeam/greptimedb/pull/5628/commits/4467745e80f468e98ea349aceb86382b8dd94cff)